cmd/app/start/start.go

```diff
@@ -89,183 +89,183 @@ func (s *Start) CreateStartCommand() *cobra.Command {
 		)
 
 	startCmd.PersistentFlags().
 		Int64P(
 			"analysis-timeout", "t",
 			s.configs.TimeoutInSecondsAnalysis,
-			"The timeout threshold for the Horusec CLI wait for the analysis to complete.",
+			"The timeout threshold for the Horusec CLI wait for the analysis to complete. The minimum time is 10",
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"authorization", "a",
 			s.configs.RepositoryAuthorization,
-			"The authorization token for the Horusec API",
+			"Authorization token to use on Horusec server. Read more: https://docs.horusec.io/docs/tutorials/how-to-create-an-authorization-token",
 		)
 
 	startCmd.PersistentFlags().
 		StringToString(
 			"headers",
 			s.configs.Headers,
-			"The headers dynamic to send on request in Horusec API. Example --headers=\"{\"X-Auth-Service\": \"my-value\"}\"",
+			`Custom headers to send on request to Horusec API. Example --headers='{"X-Auth-Service": "value"}'`,
 		)
 
 	startCmd.PersistentFlags().
 		BoolP(
 			"return-error", "e",
 			s.configs.ReturnErrorIfFoundVulnerability,
-			"The return-error is the option to check if you can return \"exit(1)\" if found vulnerabilities. Example -e=\"true\"",
+			`Return exit code 1 if found vulnerabilities. Default value is false (exit code 0)`,
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"project-path", "p",
 			s.configs.ProjectPath,
-			"Path to run an analysis in your project",
+			"Path to run an analysis. If this value is not passed, Horusec will ask if you want to run the analysis in the current directory",
 		)
 
 	startCmd.PersistentFlags().
 		Bool(
 			"enable-git-history",
 			s.configs.EnableGitHistoryAnalysis,
-			"When this value is \"true\" we will run tool gitleaks and search vulnerability in all git history of the project. Example --enable-git-history=\"true\"",
+			`Run Gitleaks and search for vulnerabilities in all git history of the project https://github.com/zricethezav/gitleaks`,
 		)
 
 	startCmd.PersistentFlags().
 		BoolP(
 			"insecure-skip-verify", "S",
 			s.configs.CertInsecureSkipVerify,
-			"Insecure skip verify cert authority. PLEASE, try not to use it. Example -S=\"true\"",
+			"Disable the certification validation. PLEASE, try not to use it",
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"certificate-path", "C",
 			s.configs.CertPath,
-			"Path to certificate of authority. Example -C=\"/example/ca.crt\"",
+			`Path to certificate of authority. Example -C="example/ca.crt"`,
 		)
 
 	startCmd.PersistentFlags().
 		BoolP(
 			"enable-commit-author", "G",
 			s.configs.EnableCommitAuthor,
-			"Used to enable or disable search with vulnerability author. Example -G=\"true\"",
+			"Enable to search commit author of vulnerabilities",
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"repository-name", "n",
 			s.configs.RepositoryName,
-			"Used to send repository name to horus server. Example -n=\"horus\"",
+			`Send repository name to Horusec server`,
 		)
```

Comment:
If using the workspace token and the repository does not exist, the name passed in the flag will be used to create the repository in the manager. If no name is given, the name of the current directory where the analysis is being performed will be the name of the repository.

I believe it is important to add the default value information. Maybe something like this:

`Send repository name to Horusec server, by default sends the actual directory name `.

Reply:
Fixed